### PR TITLE
completed dom-manipulation - brighitte johnson

### PR DESCRIPTION
**Pull request by brij8**
dom-manipulation

### CODE REVIEW

File: dom-manipulation/main.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+var $clicks = 0;
+var $hotbutt = document.querySelector('.hot-button');
+var $clickcount = document.querySelector('.click-count');
+
+$hotbutt.addEventListener('click', function (event) {
+  $clicks++;
+  $clickcount.textContent = $clicks;
+  if ($clicks < 4) {
+    $hotbutt.className = 'hot-button cold';
+  } else {
```
Comment (Zhiwutian) [inline dom-manipulation/main.js:10]: this will work, but a series of `else-if` statements would be better!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update requested!

**Comment by brij8** [comment]
that should do it

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**